pcdsdevices/ui/AttenuatorSXR_Ladder.detailed.ui

```diff
@@ -1 +1 @@
-detailed_tree.ui
\ No newline at end of file
+detailed_tree.ui
```

Comment:
What happened here?  Seems like a line was deleted then replaced?

Reply:
I think it's the newline at EOF:
```
diff --git a/pcdsdevices/ui/AT2L0.detailed.ui b/pcdsdevices/ui/AT2L0.detailed.ui
index 0711737e06..19e4afe071 120000
--- a/pcdsdevices/ui/AT2L0.detailed.ui
+++ b/pcdsdevices/ui/AT2L0.detailed.ui
@@ -1 +1 @@
-detailed_tree.ui
\ No newline at end of file
+detailed_tree.ui
```

Glad you pointed this out. It appears that pre-commit messed up symlinks here!
<img width="782" alt="image" src="https://user-images.githubusercontent.com/5139267/165345944-78ce9636-edb9-4824-92ca-9e4509354324.png">

Excluding `*.ui` from the EOL fixer seems like it could be an okay approach. Or there is a "types" option in pre-commit `symlinks` that could be used to exclude it somehow, it appears.

Reply:
I think excluding .ui and also excluding symlinks will both be necessary. I think these should be excluded from all checks for this repo, since .ui is always generated from designer and therefore there isn't any particular style consideration to figure out. The symlink thing is weird.

Reply:
RE: types, we would need to exclude symbolic links on a per-hook basis. For now I'll just slap this onto the EOL fixer.